sdk/dotnet/ContainerService/GetKubernetesCluster.cs

```diff
@@ -52,15 +52,6 @@ public sealed class GetKubernetesClusterArgs : Pulumi.InvokeArgs
         [Input("name", required: true)]
         public string Name { get; set; } = null!;
 
-        /// <summary>
-        /// If the cluster has the Kubernetes API only exposed on internal IP addresses.                           
-        /// </summary>
-        [Input("privateClusterEnabled")]
-        public bool? PrivateClusterEnabled { get; set; }
-
-        [Input("privateLinkEnabled")]
-        public bool? PrivateLinkEnabled { get; set; }
```

Comment:
That looks like a breaking change, what happened to these two?

Reply:
According to the CHANGELOG in AzureRM Provider:

```
Data Source: azurerm_kubernetes_cluster
bug fix: fixing an issue where some read-only fields were unintentionally marked as user-configurable.
```

These are those specific fields!

https://github.com/terraform-providers/terraform-provider-azurerm/pull/7233/files#diff-ceb52ca7b573864a4c727e76b6e4c09cL234